repo-addrepo-hd-iso.ks.in

```diff
@@ -0,0 +1,42 @@
+#test name: repo-addrepo-hd-iso
+
+## Test the inst.addrepo boot option with an ISO on a hard drive.
+
+# This test is resource demanding! It downloads a full DVD ISO.
```

Comment:
Reading just the comment, I thought it would need bandwidth for the download. After reading the code, I think it's demanding on the VM storage resources. So, perhaps make the comment more specific?